src/tools/shapes/RulerCreator.ts

```diff
@@ -0,0 +1,111 @@
+import Konva from 'konva'
+import { CanvasElement } from '@/types/Canvas'
+import { Shape } from '@/tools/shapes/Shape'
+
+export default class RulerCreator implements Shape {
+  private line: Konva.Line
+  private text: Konva.Text
+  private circle: Konva.Circle
+  private readonly mapRatio: number
+  constructor (public size: number,
+               public colour: string) {
+    this.line = new Konva.Line()
+    this.text = new Konva.Text()
+    this.circle = new Konva.Circle()
+    this.mapRatio = 1
+  }
+
+  create = (canvasElement: CanvasElement, layer: Konva.Layer): CreateRuler => {
+    this.line = this.createLineElement(canvasElement)
+    this.text = this.createTextElement(canvasElement)
+    this.circle = this.createCircleElement(canvasElement)
+    layer.add(this.circle)
+    layer.add(this.text)
+    layer.add(this.line)
+    return {
+      circle: this.circle,
+      line: this.line,
+      text: this.text
+    }
+  }
+
+  // eslint-disable-next-line
```

Comment:
what is eslint complaining about here btw?

Reply:
Probably some old code from before the rewrite, will remove